CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Unreleased
 
+## 0.7.4
+### January 4, 2024
+
+IMPROVEMENTS:
+* update dependencies
```

Comment:
nit: should we also add which dependencies were updated?

Reply:
oh sure! done~